Solutions/SophosEP/Parsers/SophosEPEvent.txt

```diff
@@ -0,0 +1,24 @@
+// Usage Instruction : 
+// Paste below query in log analytics, click on Save button and select as Function from drop down by specifying function name and alias as SophosEPEvent.
+// Function usually takes 10-15 minutes to activate. You can then use function alias from any other queries (e.g. SophosEPEvent | take 10).
+// Reference : Using functions in Azure monitor log queries : https://docs.microsoft.com/azure/azure-monitor/log-query/functions
+SophosEP_CL
+| extend EventVendor = 'Sophos'
+| extend EventProduct = 'Endpoint Protection'
+| project-rename DstUserSid=user_id_s,
+    CustomerId=customer_id_g,
+    EventSeverity=severity_s,
+    Created=created_at_t,
+    SrcIpAddr=source_info_ip_s,
+    ThreatName=threat_s,
+    EndpointId=endpoint_id_g,
+    DvcType=endpoint_type_s,
```

Comment:
It should be SrcDvcType. Plesae refer the  naming convention from [here](https://docs.microsoft.com/en-us/azure/sentinel/normalization-schema).

Thanks,
Aditi Maheshwari

Reply:
updated